packages/management/framework/bundle/config/pacakgeinstalls/addons-manager-pi.yaml

```diff
@@ -0,0 +1,33 @@
+#@ load("@ytt:yaml", "yaml")
+#@ load("@ytt:data", "data")
+
+---
+apiVersion: packaging.carvel.dev/v1alpha1
+kind: PackageInstall
+metadata:
+  name: tanzu-addons-manager
+  namespace: #@ data.values.namespaceForPackageInstallation
+  annotations:
+    kapp.k14s.io/change-rule.0: "upsert after upserting addons-manager-packageinstall/serviceaccount"
+    kapp.k14s.io/change-rule.1: "delete before deleting addons-manager-packageinstall/serviceaccount"
+spec:
+  serviceAccountName: tanzu-addons-manager-package-sa
+  packageRef:
+    refName: addons-manager.tanzu.vmware.com #! TODO: confirm the addons manager package refName in uTKG
```

Comment:
Is the package name addons-manager.tanzu.vmware.com or addons-manager-v2.tanzu.vmware.com ?

Reply:
by looking at [this](https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/management/addons-manager-v2/package.yaml#L7), it should be `addons-manager.tanzu.vmware.com`

Reply:
I think we will drop the v2 name going forward. The current v2 name is only meant to be added to avoid conflict in 1.6.0 release.

Reply:
> by looking at [this](https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/management/addons-manager-v2/package.yaml#L7), it should be addons-manager.tanzu.vmware.com

When I tried building locally, I was able to build the package repository but it felt like the available packages are based on contains v2. Here I have installed core as well as management package repository.
```
tkg-system   addons-manager-v2.tanzu.vmware.com.0.24.0-dev                                        addons-manager-v2.tanzu.vmware.com                   0.24.0-dev                                 9m1s
tkg-system   addons-manager.tanzu.vmware.com.1.5.0+vmware.1-tkg.2-zshippable                      addons-manager.tanzu.vmware.com                      1.5.0+vmware.1-tkg.2-zshippable            4h33m9s
```

I am suspecting when building locally, the package name relies on https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/package-values.yaml#L43 ? cc @yharish991

Reply:
Got the management repo bundle from our downstream build process and checked the package name. It is addons-manager-v2. Please ensure package name is correct

Reply:
Looks like this file https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/management/addons-manager-v2/package.yaml#L7 also needs to be updated.

Reply:
Updated both files to v2